elyra/pipeline/pipeline_definition.py

```diff
@@ -188,6 +192,22 @@ def set_property(self, key: str, value: Any):
 
         self._node["app_data"]["properties"][key] = value
 
+    def convert_kv_pair_properties(self, kv_pair_properties: List[str]):
```

Comment:
Would you mind reducing `kv_pair` to `kv` throughout?  kv-pair seems a bit redundant since key-value is the pair.
```suggestion
    def convert_kv_properties(self, kv_pair_properties: List[str]):
```